README.md

```diff
@@ -22,80 +22,47 @@ Much like the contents of a container image need to be pulled and unpacked in or
+### BundleInstance 
+The `BundleInstance` API points to a Bundle and indicates that it should be “active”. This includes pivoting from older versions of an active bundle. 
+`BundleInstance` may also include an embedded spec for a desired Bundle.
 
-Much like Pods stamp out instances of container images, `Instances` stamp out an instance of Bundles. `Instance` can be seen as a generalization of the Pod concept.
+Much like Pods stamp out instances of container images, `BundleInstances` stamp out an instance of Bundles. `BundleInstance` can be seen as a generalization of the Pod concept.
 
-The specifics of how an `Instance` makes changes to a cluster based on a referenced `Bundle` is defined by the 
+The specifics of how an `BundleInstance` makes changes to a cluster based on a referenced `Bundle` is defined by the 
 `ProvisionerClass` and the `Provisioner` that is configured to handle that `ProvisionerClass`.
 
-Example Instance
+Example BundleInstance
 ```yaml
 apiVersion: core.rukpak.io/v1alpha1
-kind: Instance
+kind: BundleInstance
 metadata:
-  name: resolved-654adh
+  name: combo-system
 spec:
-  selector: 
-    matchLabels:
-      subscription: etcd-operator
-  bundle:
-    name: resolved-654adh
-    spec:
-      class: rukpack.io/k8s
-      refs:
-      - file://content
-      volumeMounts:
-      - mountPath: /content
-        configMap:
-          name: resolved-654adh-content
-          namespace: olm
+  provisionerClassName: core.rukpak.io/plain
+  bundleName: combo-0.0.1
 ```
 
-### ProvisionerClass
-`ProvisionerClass` defines a configuration for a `Provisioner`. Provisioners are controllers that understand `Instance`, `Bundle`, and `ProvisionerClass` APIs and take action.
-
-Each `Provisioner` has a unique id. For example, a `Provisioner` that understands bundles composed of arbitrary Kubernetes YAML manifests can have the id: rukpack.io/k8s.
-
-A `ProvisionerClass` specifies a specific configuration of provisioning (i.e. settings to interpret Instance and Bundle APIs). 
-A `Provisioner` will only operate on Instance and Bundle objects that reference a `ProvisionerClass` that contain the provisioner’s unique id.
-
-If this seems familiar, it is the same pattern that is used by StorageClass / PeristentVolume / PersistentVolumeClaim in Kubernetes. This design is meant to be extendable 
-to support `Bundles` and `Instances` backed by different content sources. 
+### Provisioner
+A Provisioner is a controller that understands `BundleInstance` and `Bundle` APIs and can take action. Each `Provisioner` is assigned a unique ID, and is responsible for reconciling
+a `Bundle` and `BundleInstance` with a `spec.provisionerClassName` that matches that particular ID. For example, in this repository the [plain k8s](provisioner/k8s/README.md) provisioner is implemented. 
+The `k8s` provisioner is able to unpack a given `k8s` bundle onto a cluster and then instantiate it, making the
+content of the bundle available in the cluster. 
```

Comment:
super nit: adding new lines to README's makes it weird to look at on smaller screens.

```suggestion
A Provisioner is a controller that understands `BundleInstance` and `Bundle` APIs and can take action. Each `Provisioner` is assigned a unique ID, and is responsible for reconciling a `Bundle` and `BundleInstance` with a `spec.provisionerClassName` that matches that particular ID. For example, in this repository the [plain k8s](provisioner/k8s/README.md) provisioner is implemented. The `k8s` provisioner is able to unpack a given `k8s` bundle onto a cluster and then instantiate it, making the content of the bundle available in the cluster. 
```

Reply:
Whatever we chose, at least be consistent in the file. I wonder if there's a markdown linter that we can use for something like that?

fwiw, my preference is newlines, but with a strict column cutoff (e.g. 100 chars), but i fully realize this is like a tabs vs spaces flamewar, and I'm not interested in starting one here :rofl:

Reply:
I looked for a markdown linter that we can add to our existing lint commands but didn't find one. I ended up just using the markdown linter in my IDE. Hopefully that will address any concerns for now.